iaso/migrations/0345_merge_20250926_1109.py

```diff
@@ -0,0 +1,12 @@
+# Generated by Django 4.2.20 on 2025-09-26 11:09
+
+from django.db import migrations
+
+
+class Migration(migrations.Migration):
+    dependencies = [
+        ("iaso", "0344_alter_corepermissionsupport_options"),
+        ("iaso", "0344_planning_pipeline_uuids"),
+    ]
+
+    operations = []
```

Comment:
Why don't you just rename your migration to `0345` and point to the previous `0344` for the dependency?

Reply:
you are right, i always did it like that